menu.html

```diff
@@ -29,21 +29,27 @@
 
 
 
-          <li class="navbar-brand ">
+    <li class="navbar-brand ">
 	    <a class="nav-link active" href="projects.html">Projects</a>
-	  </li>
+	</li>
 
           
           
 <!---  DEBUT Menu Events & Positions  --->
-	  <li class="navbar-brand dropdown active">
+
+	 <!-- Positions removed
+		<li class="navbar-brand dropdown active">
 			<a class="nav-link dropdown-toggle item" data-target="events.html"  href="events.html"   
 				   id="dropdown01" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Events and Positions</a>
 			<div class="dropdown-menu" aria-labelledby="dropdown01">
 			  <a class="dropdown-item" href="events.html">Events</a>
 			  <a class="dropdown-item" href="announce.html">Annoucements for positions</a> 
```

Comment:
You remove the announce position page ? Less and less use perhaps not needed outside to see archive.

Reply:
Yes, the only open position was probably largely outdated (2 years olds). We could think about reestablishing the page if we find a way to make people use it